recipes/pyibs/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyibs" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyIBS-{{ version }}.tar.gz
```

Comment:
Personally I prefer the GitHub source as it is closer to the source, increasing the changes not running out-of-date. But if you are the maintainer of the library it could be fine if you know for sure that the upload to pypi is automatic

Reply:
And I'm the maintainer of the library, is that fine?

Reply:
Yeah sure!

Reply:
Hi! It's been a few days. Should I make any changes to get my package approved?

Reply:
I think that there is a shortage of people with push-rights (that I  don't have) that makes that things takes time. You can try to write on the gitter